tests/integrationv2/test_key_update.py

```diff
@@ -28,8 +28,8 @@ def test_s2n_server_key_update(managed_process, cipher, provider, other_provider
     port = next(available_ports)
 
     update_requested = b"K"
-    server_data = data_bytes(10)
-    client_data = data_bytes(10)
+    server_data = "Some data from the server!"
+    client_data = "Some data from the client!"
```

Comment:
I guess that's better :/ But why can't you use random data? Doesn't every other test use random data?

Like, just looking at the first integ test on our list, it's successfully using data_bytes: https://github.com/aws/s2n-tls/blob/main/tests/integrationv2/test_buffered_send.py#L94-L105 Although also looking at that test, maybe close_marker is what's giving you issues? Looks like that test adds a fixed suffix specifically so that it doesn't have to use random bytes as the close_marker.

Maybe just give up on random bytes, but still use random text? Like https://stackoverflow.com/questions/2257441/random-string-generation-with-upper-case-letters-and-digits. Just append a short random sequence to "client data: ".